app/services/spree_stripe/create_source.rb

```diff
@@ -55,27 +53,27 @@ def find_or_create_credit_card
     end
 
     def credit_card_params
-      card_details = stripe_charge.payment_method_details.card
-      customer = gateway.fetch_or_create_customer(order: order, user: user)
```

Comment:
We user `bill_address` from the `order` record when creating the Stripe Customer. Is this not needed anymore?

Reply:
Hmm, at this point `bill_address` should be assigned to the `user` as well, but I'll re-add the `order` param

Reply:
I think it's done for guest orders - including quick checkout orders.